src/go/rpk/pkg/api/admin/api_features.go

```diff
@@ -47,7 +47,7 @@ type LicenseProperties struct {
 	Version      int    `json:"format_version"`
 	Organization string `json:"org"`
 	Type         string `json:"type"`
-	Expires      int    `json:"expires"`
+	Expires      int64  `json:"expires"`
```

Comment:
```suggestion
	Expires      int64  `json:"expiresAt"`
```

Reply:
Making this change would reflect as a modification in the json key, the standard is for the c++ code to use `snake_case` for all keys